pom.xml

```diff
@@ -34,6 +34,22 @@
          </exclusions>
       </dependency>
 
+      <dependency>
+         <groupId>org.apache.solr</groupId>
+         <artifactId>solr-solrj-zookeeper</artifactId>
+         <version>9.1.0</version>
+      </dependency>
+      <dependency>
+         <groupId>org.apache.zookeeper</groupId>
+         <artifactId>zookeeper</artifactId>
+         <version>3.8.0</version>
+      </dependency>
+      <dependency>
+         <groupId>org.eclipse.jetty</groupId>
+         <artifactId>jetty-client</artifactId>
+         <version>9.4.48.v20220622</version>
```

Comment:
any particular reason to stick with 9.4.48 version?

Reply:
O I used the runtime dependency version of solrj 9.1.0 :) 
https://mvnrepository.com/artifact/org.apache.solr/solr-solrj/9.1.0